README.md

```diff
@@ -35,124 +35,36 @@ AKS Periscope currently only work on Linux based agent nodes. Please use the scr
 
 It collects the following logs and metrics:
 
-1. Container logs (by default all containers in the `kube-system` namespace, can be config to take other namespace/containers).
-2. Docker and Kubelet system service logs.
-3. Network outbound connectivity, include checks for internet, API server, Tunnel, Azure Container Registry and Microsoft Container Registry.
-4. Node IP tables.
-5. All node level logs (by default cluster provision log and cloud init log, can be config to take other logs).
-6. VM and Kubernetes cluster level DNS settings.
-7. Describe Kubernetes objects (by default all pods/services/deployments in the `kube-system` namespace, can be config to take other namespace/objects).
-8. Kubelet command arguments.
-9. System performance (kubectl top nodes and kubectl top pods).
+1. Network outbound connectivity, include checks for internet, API server, Tunnel, Azure Container Registry and Microsoft Container Registry.
+1. Node IP tables.
+1. All node level logs (by default cluster provision log and cloud init log, can be config to take other logs).
+1. VM and Kubernetes cluster level DNS settings.
+1. Describe Kubernetes objects (by default all pods/services/deployments in the `kube-system` namespace, can be config to take other namespace/objects).
+1. Kubelet command arguments.
+1. System performance (kubectl top nodes and kubectl top pods).
+1. Helm charts status
+1. SMI & OSM deployments information
 
 It also generates the following diagnostic signals:
 
 1. Network outbound connectivity, reports the down period for a specific connection.
-2. Network configuration, includes Network Plugin, DNS, and Max Pods per Node settings.
+1. Network configuration, includes Network Plugin, DNS, and Max Pods per Node settings.
 
 ## User Guide
 
-AKS Periscope can be deployed by using Azure Command-Line tool (CLI). The steps are:
+AKS Periscope can be deployed by using Azure Command-Line tool (CLI). You will have to install the `aks-preview` extension to do so.  
+See [az cli aks-preview extension](https://github.com/Azure/azure-cli-extensions/tree/main/src/aks-preview/azext_aks_preview) for detailed information.
 
-0. If CLI extension aks-preview has been installed previously, uninstall it first.
```

Comment:
🐛 Why is this removed? Reason? (Like mentioned few times primary contract of this tool is to coexist with `az-cli` and `vscode` so lets sync up a meeting with @qpetraroia and @rzhang628 - but we cannot remove this, `Collect` command is one of the key use for this and supports few AKS scenarios.

Reply:
yes but the kollect doc shall be in the kollect repo. This documentation should be moved. It's not aks-periscope reponsability to know how external tools use it.
Furthermore, to maintain this documentation up-to-date it would mean to watch for new releases of kollect command and update our readme accordingly - this is painful

Reply:
💡 I have blocked some time this week, might be there is missing information or could be background information, hence you probably are making these changes, the `collect` command was written at the same time and was the idea behind this tool. Hence in various places including the internals have updated a readMe file regarding consuming tools here: https://github.com/Azure/aks-periscope#dependent-consuming-tools-and-working-contract 

I would recommend that we can improve the documentation by proper - Wiki pages. Which might also include the consensus on other things. Removing the key usability docs from the documentation is a bad idea. We can discuss this in tomorrows catch-up (if possible) leaving this for wider PM visibility.

Reply:
Don't you think it's weird to have the code in one repo and documentation in another one, managed by another team? That's why I suggested to put code & doc at the same place. If we agree on this PR all doc we remove from here, we put it in a PR on the kollect repo and add a link to this repo from our readme. In my opinion, that would make more sense.

Reply:
Cool, @rzhang628 - has an idea for workitem to handle this in `wiki` for this repo. Which sounds like a great plan and thanks for that.